pkg/reconciler/taskrun/taskrun.go

```diff
@@ -418,6 +418,10 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1beta1.TaskRun, rtr *re
 	logger := logging.FromContext(ctx)
 	recorder := controller.GetEventRecorder(ctx)
 
+	if tr.Spec.ComputeResources != nil {
```

Comment:
could you please add validation that if a taskrun has compute resources, the task spec does not specify step-level resources?

Reply:
resolved - by adding the validation